packages/@nucleus/tests/dummy/app/templates/components/playground/props.hbs

```diff
@@ -0,0 +1,45 @@
+{{#each properties as |property|}}
+  <div class="item">
+    <label for="{{property.name}}" class="item--label">
+      {{property.name}}
+    </label>
+
+    {{!-- select box --}}
+    {{#if property.select}}
+      <select
+        onchange={{action 'handleChange' property}}
+        name="{{property.name}}"
+        id="{{property.name}}"
+        value="{{property.value}}">
+        {{#each property.types as |type|}}
+          <option value="{{type}}">{{type}}</option>
+        {{/each}}
+      </select>
+    {{/if}}
+
+    {{!-- input --}}
+    {{#if property.input}}
```

Comment:
A single property cannot have both select and input flags right? In that case, can we refactor this to something like `property.type` and switch based on that?